kuma/wiki/templates/wiki/document.html

```diff
@@ -58,8 +59,8 @@
   <link rel="alternate" type="application/json" href="{{ url('wiki.json_slug', document.slug) }}">
   <link rel="canonical" href="{{ canonical }}" >
 
-  {% if is_translated %}
-    {% for translation in document.other_translations %}
+  {% if other_translations %}
```

Comment:
Wait, you're right, `other_translations` is a cached property of the document object and *not* a related manager as I thought. So forget what I said.